cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -1023,6 +1023,19 @@ def load_resource(
         transformations = TransformationWriteList([])
 
         for resource in resources:
+            external_id = resource.get("externalId", "Missing")
```

Comment:
Perhaps make it even more clear, I leave it up to you:
```suggestion
            external_id = resource.get("externalId", "<Missing>")
```